retroshare-gui/src/gui/chat/ChatLobbyDialog.cpp

```diff
@@ -411,7 +435,7 @@ void ChatLobbyDialog::init(const ChatId &/*id*/, const QString &/*title*/)
         if(rsIdentity->getIdDetails(gxs_id,details))
             break ;
         else
-            rstime::rs_usleep(1000*300) ;
+            std::this_thread::sleep_for(std::chrono::microseconds(1000*300)) ;
```

Comment:
you can just use `milliseconds(300)` instead

Reply:
Sure